docs/terminal/amberleyoakey.md

```diff
@@ -69,31 +69,64 @@ OK ADC owns the Class C airspace within 5 DME of the OK VOR from `SFC` to `A025`
 
+Aircraft will depart the circuit visually and track overhead YAMB to begin tracking for their initial waypoint.
 
-Aircraft will make a visual departure in the circuit direction and fly overhead YAMB to begin tracking for their initial waypoint. 
+!!! phraseology
+    WOLF03 was assiged the BYRON1 departure with their initial clearance.  
+    **AMB ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
+    **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  
 
 | Departure | Initial Constraint | Route |
 | --------- | ----------| --------- |
-| Northern 1 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
-| Central 1 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
-| Byron 7 | `F190` at COWIE | COWIE KIWEE ZANET LOTMA SEMAJ |
-
-!!! phraseology
-    WOLF03 was assiged the BYRON7 departure with their intial clearance.  
-    **AMB ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
-    **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"
+| Northern 3 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
+| Northern 4 | `F200` at ANKED | ANKED JEZZA BOBED LESKO MOSSI/ADNUK (Transition) |
+| Central 3 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
+| Central 4 | `F200` at ANKED | ANKED JEZZA BOBED VIRGE |
+| Byron 1 | `F190` at COWIE | COWIE KIWEE ZANET LOTMA SEMAJ |
+| Byron 2 | `F200` at TUGUN | TUGUN LAGOB ZANET LOTMA SEMAJ |
 
 !!! note
-    AMB ACD will record the name of the coded departure in the Global Ops Field of the FDR and amend the flight plan route with all necessary tracking points. 
+    Since the Coded Departures are not SIDs, AMB ACD needs to write the name of the coded departure in the Global Ops Field of the FDR and amend the flight plan route to include all relevant tracking points.
+
+!!! abstract "Reference"
+    Charts for the Coded Departures are available in: [YAMB FIHA AD2 Supp Section 6.2.25](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+## Civil Operations
+Civil and AAFC conduct flying operations within the South Western Training Area (SWTA). The SWTA is divided in half by the Rosewood-Mt Walker-Aratula Road, SWTA Alpha (north) and SWTA Bravo (south). 
+
+Once established in the SWTA, aircraft will remain OCTA. Pilots may request to enter the CTA and climb above `A025` when operationally required.
+
+**Departures**:
+
+| **Duty Runway** | **Routing** | **Altitude** |
+|-----------------|-----------|--------------|
+| 15              | MTWK      | `A025`         |
+| 33              | CLVT      | `A025`        |
+
+**Arrivals**:
+
+| **Duty Runway** | **Routing** | **Altitude**         |
+|-----------------|-----------|----------------------|
+| 15              | CLVT      | `A025` or `A015`        |
+| 33              | MTWK      | `A025` or `A015`       |
```

Comment:
Any guidance on which level to choose?

Reply:
Aircraft are always cleared 1500ft when departing, thence remaining OCTA once within the SWTA. 

However, if an aircraft operating in SWTA needs to climb into CTA (above 2500ft) for operations, Amberley Approach can instruct "maintain A025" on its return to base instead of descending to 1500ft, as it was already operating at that altitude. Thence once with ADC overhead the inbound VFR waypoint, they can descend and make the approach.

Reply:
Cheers

Reply:
Wait A015 or A025 outbound? You've got A025 in the doc

Reply:
Oh yeah you're right. just doubled checked the sup and my writing and departures are at A025. In that case i'd assume it would be TCU's discretion for the level? Aircraft are OCTA within SWTA so before exiting could have finished operations at A015 possibly, and call approach for airways clearance at A015.

Reply:
Cool. I'm almost finished revising this stuff, will push shortly.

Reply:
awesome thanks